flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveTableSinkITCase.java

```diff
@@ -153,15 +161,15 @@ public void testBatchAppend() throws Exception {
 
     @Test
     public void testDefaultSerPartStreamingWrite() throws Exception {
-        testStreamingWrite(true, false, "textfile", this::checkSuccessFiles);
+        testStreamingWrite(true, false, "textfile", this::checkPartitionCommitSuccessfully);
```

Comment:
IMO, the original check for success files doesn' need to modify. You could add the new tests for the custom `PartitionCommitPolicy`.